### DISCUSSION TITLE
Aniyomi update

### DISCUSSION THREAD

**Discussion by Raven0770** [aniyomiorg/aniyomi#738] [General]
When are we getting stable version of aniyomi and when will they resolve extension problem of 9Anime and many other extension,......?

**Comment by ImpenetrableNoble** [comment]
Read the FAQ in the Aniyomi discord server the question has already been answered. 

Sent from miss moriarty  
 
  On Thu, Aug 25, 2022 at 8:00 AM, ***@***.***> wrote:   


When are we getting stable version of aniyomi and when will they resolve extension problem of 9Anime and many other extension,......?
